plugins/periskop/src/plugin.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  discoveryApiRef,
+  configApiRef,
+  createApiFactory,
+  createPlugin,
+  createApiRef,
+} from '@backstage/core-plugin-api';
+import { PeriskopApi } from './api';
+
+/**
+ * @public
+ */
+export const periskopApiRef = createApiRef<PeriskopApi>({
```

Comment:
It's best if the API has its own dedicated type that is not the concrete implementation. So in this case perhaps converting `PeriskopApi` to an `interface`, and calling the concrete implementation `PeriskopClient`

Reply:
Thanks. Done in [`5afe204` (#9816)](https://github.com/backstage/backstage/pull/9816/commits/5afe2046db8e311600c949a7ce7b92e48694872b)